lib/module.js

```diff
@@ -460,29 +461,23 @@ Module.runMain = function() {
 };
 
 Module._initPaths = function() {
-  const isWindows = process.platform === 'win32';
+  modulePaths = [];
```

Comment:
Yes. `modulePaths` is global to this module, and calling `_initPaths`, resets its value. https://github.com/nodejs/node/blob/810cc05116b39963b25c87fd17be32a6bb867eb0/lib/module.js#L485